homeassistant/components/airvisual/sensor.py

```diff
@@ -59,60 +59,82 @@
 SENSOR_KIND_TEMPERATURE = "temperature"
 SENSOR_KIND_VOC = "voc"
 
-GEOGRAPHY_SENSORS = [
-    (SENSOR_KIND_LEVEL, "Air Pollution Level", "mdi:gauge", None),
-    (SENSOR_KIND_AQI, "Air Quality Index", "mdi:chart-line", "AQI"),
-    (SENSOR_KIND_POLLUTANT, "Main Pollutant", "mdi:chemical-weapon", None),
-]
+GEOGRAPHY_SENSOR_DESCRIPTIONS = (
```

Comment:
```suggestion
GEOGRAPHY_SENSOR_DESCRIPTIONS: tuple[SensorEntityDescription, ...] = (
```

Reply:
Why? `mypy` doesn't complain without this, which means it can infer the type.

Reply:
It's a nice to have, but yes not strictly necessary.